src/modules/market/selectors/market.js

```diff
@@ -291,7 +291,7 @@ export function assembleMarket(
 
       market.tags = (market.tags || []).map((tag) => {
         const obj = {
-          name: tag && tag.toString().toLowerCase().trim(),
+          name: tag && tag.toString().trim(),
```

Comment:
My feeling is that we should display the tags as-is, even if they include stupid leading/trailing whitespace.  Presumably people will refrain from doing that b/c it will make their tag unique and dumb, and very far down the popularity list!